applications/crossbar/src/modules/cb_rates.erl

```diff
@@ -133,6 +150,23 @@ put(Context) ->
 delete(Context, _RateId) ->
     crossbar_doc:delete(Context).
 
+-spec validate_number(ne_binary(), cb_context:context()) -> cb_context:context().
+validate_number(Phonenumber, Context) ->
+    case length(re:replace(Phonenumber, "[^0-9]", "", [global, {return, list}])) >= 7 of 
```

Comment:
I think a better test is `wnm_util:is_reconcilable/1`. A number that is considered "reconcilable" is a "global" number for routing purposes, which is when rates would be applied.